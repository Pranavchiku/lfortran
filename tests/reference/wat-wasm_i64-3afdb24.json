{
    "basename": "wat-wasm_i64-3afdb24",
    "cmd": "lfortran --no-color --show-wat {infile}",
    "infile": "tests/wasm_i64.f90",
    "infile_hash": "9793bf432037540bb08d6e1429b2967f40f93c61e08901b587d2df2d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-wasm_i64-3afdb24.stdout",
    "stdout_hash": "8de9fc9d6f0ebb745912cb0f0b558a51d0a7f6a0562128affbcb608d",
    "stderr": "wat-wasm_i64-3afdb24.stderr",
    "stderr_hash": "b9a66efab51340bf9c5b80a20d79521dd72e1025217d55cc44d6d1f6",
    "returncode": 0
}