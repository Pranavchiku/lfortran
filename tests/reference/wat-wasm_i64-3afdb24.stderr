warning: Implied save feature
  --> tests/wasm_i64.f90:12:9
   |
12 |         integer(8), intent(in):: x
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:13:9
   |
13 |         integer(8) :: r
   |         ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:20:9
   |
20 |         integer(8), intent(in):: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:20:9
   |
20 |         integer(8), intent(in):: x, y
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:21:9
   |
21 |         integer(8) :: r
   |         ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:28:9
   |
28 |         integer(8) :: x64, y64
   |         ^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:28:9
   |
28 |         integer(8) :: x64, y64
   |         ^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/wasm_i64.f90:29:9
   |
29 |         integer(8) :: r
   |         ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
